app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -58,6 +69,8 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler {
         setupNav()
 
         handleNotifications(intent)
+
+        (application as? HeimdallApplication)?.registerForActivity(this)
```

Comment:
should we move it to a base activity? (OnActivityChanged could be also implemented by a base activity)

Reply:
Navigation.findNavController() doesnt't work with the BaseActivity. Or I don't know how. It seems to need `this@StartActivity`